lua/entities/gmod_wire_expression2/core/glon.lua

```diff
@@ -288,7 +322,7 @@ end
 
 --- Decodes <data> into an array, using [[von]].
 e2function array vonDecode(string data)
-	if not data then return {} end
+	if not data or data == "" then return {} end
```

Comment:
No, I didn't think there were nil strings either, which is why I added that other check just now. But everything has worked up until now so I don't think it causes very many problems either way.